uzipoc-cibg/Issues/uzi/description.xml

```diff
@@ -20,27 +20,57 @@
 	</IssueURL>
 
 	<Attributes>
-		<Attribute id="uraName">
+		<Attribute id="initials">
+			<Name>
+				<en>Initials</en>
+				<nl>Initialen</nl>
+			</Name>
+			<Description>
+				<en>Your initials</en>
+				<nl>Uw initialen</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="surname_prefix">
+			<Name>
+				<en>Surname prefix</en>
+				<nl>Tussenvoegsel</nl>
+			</Name>
+			<Description>
+				<en>Your surname prefix</en>
+				<nl>Uw tussenvoegsel</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="surname">
+			<Name>
+				<en>Surname</en>
+				<nl>Achternaam</nl>
+			</Name>
+			<Description>
+				<en>Your surname</en>
+				<nl>Uw achternaam</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="entity_name">
```

Comment:
An existing credential cannot be modified retrospectively. IRMA apps that already contain this credential will get conflicts then. Either a fully new credential should be added instead or the new attributes should be added at the end of the `Attributes` list as optional attributes while keeping the existing attributes.

Reply:
Because of the closed demo there are only a few people that could've loaded this credentials. So it is not a problem that we have to remove the existing credentials in this demo environment.

Reply:
I understand, but these testing credentials have been publicly usable so you never know who tested using these credentials. On top of that, the credentials are also present in the activity logs of users. Therefore, it is our policy to not remove any existing credentials. It might brick IRMA-apps of people.